README.md

```diff
@@ -140,3 +141,7 @@ This horizontal cluster proportional autoscaler is a DIY container (because it i
 There is no requirement to run heapster and/or provide CPU resource limits as in HPAs.
 
 The ConfigMap provides the operator with the ability to tune the replica scaling explicitly.
+
+## Using NodeLabels
+
+Nodelabels is an optional param to count only nodes and its cpus where the nodelabels exits. This is useful when nodeselector is used on the target pods controller so its needed to take account only the nodes tagged with the nodeselector labels to calculate the total replicas to scale. When the param is ignored then the cluster proportional autoscaler counts all schedulable nodes and its cpus (old default behaviour).
```

Comment:
It seems like counting all schedulable nodes should still be the default behavior instead of "old default behavior"?

Reply:
Yes!, 
I tried to explain that if nodeselector param is omitted then proportional-autoscaler will count all schedulable nodes without taking into account labels.

I signed the CLA, I am thinking that the problem is the github user in the commit.